local-dev/templates/rawls.conf

```diff
@@ -53,15 +45,15 @@ gcs {
   notifications.topicName=workbench-notifications-local
 
   # custom iam roles
-  requesterPaysRole=organizations/{{ $orgId }}/roles/RequesterPays
-  terraBucketReaderRole=organizations/{{ $orgId }}/roles/terraBucketReader
-  terraBucketWriterRole=organizations/{{ $orgId }}/roles/terraBucketWriter
-  terraBillingProjectOwnerRole=organizations/{{ $orgId }}/roles/terra_billing_project_owner
-  terraWorkspaceCanComputeRole=organizations/{{ $orgId }}/roles/terra_workspace_can_compute
-  terraWorkspaceNextflowRole=organizations/{{ $orgId }}/roles/terra_workspace_nextflow_role
+  requesterPaysRole=organizations/400176686919/roles/RequesterPays
+  terraBucketReaderRole=organizations/400176686919/roles/terraBucketReader
+  terraBucketWriterRole=organizations/400176686919/roles/terraBucketWriter
+  terraBillingProjectOwnerRole=organizations/400176686919/roles/terra_billing_project_owner
+  terraWorkspaceCanComputeRole=organizations/400176686919/roles/terra_workspace_can_compute
+  terraWorkspaceNextflowRole=organizations/400176686919/roles/terra_workspace_nextflow_role
 
   servicePerimeters {
-    staticProjects = {{$staticPerimeterProjects | toJSON}}
+    staticProjects = {}
```

Comment:
I think we don't need anything here for local development, see https://broadinstitute.slack.com/archives/C03BCARP494/p1727097190516919?thread_ts=1727096967.429529&cid=C03BCARP494